src/api/__tests__/studyStats.test.ts

```
@@ -0,0 +1,40 @@
+import { describe, it, expect } from 'vitest';
+import { parseStudyStats } from '../studyStats';
+
+// Real IS Mendelu labels put a non-breaking space after the single-letter
+// preposition "z" — "Průměr z odstudovaných...". Credit-count
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)